io.github.f3d_app.f3d.yaml

```diff
@@ -0,0 +1,309 @@
+app-id: io.github.f3d_app.f3d
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '21.08'
+command: f3d
+rename-desktop-file: f3d.desktop
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
```

Comment:
I see this build tcl/tk. I don't think Tk support x11, but also I'm not sure how it is used. All I know is that if Tk is used, it will need `--socket=x11` instead.

Reply:
that's true, will change to x11